test/cli/cli_test.go

```diff
@@ -115,24 +133,13 @@ func RunFunctionalTest(t *testing.T, testDir, workingDir string, binary string)
 		t.Fatalf(`Cannot parse args "%s": %s`, argsStr, err)
 	}
 
-	cmdEnvs, err := env.FromOs()
-	assert.NoError(t, err)
-	cmdEnvs.Unset(`KBC_STORAGE_API_HOST`)
-	cmdEnvs.Unset(`KBC_STORAGE_API_TOKEN`)
```

Comment:
Tu sa unsetovali niektore OS envs, to uz nie je potrebne, kedze ziadne ENVs z OS sa tam nedostanu.